core/config.py

```diff
@@ -48,7 +47,7 @@
     # Maps Domains to alternative default languages
     "viur.domainLanguageMapping": {},
     # If set, the expiration parameter for :meth:`viur.core.render.html.env.viur.downloadUrlFor` can be omitted
-    "viur.downloadUrlFor.expiration": unsetMarker,
+    "viur.downloadUrlFor.expiration": None,
```

Comment:
Can we name this `viur.render.html.downloadUrlExpiration` like for the json render and move it below?

```suggestion
    "viur.render.html.downloadUrlExpiration": None,
```

Reply:
I've introduced your request in b42622a0c32e5f5081930e976d2d8272c80e48e4